D/difmap/build_tarballs.jl

```diff
@@ -0,0 +1,48 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "difmap"
+version = v"2.5.5"
+
+sources = [
+    ArchiveSource("ftp://ftp.astro.caltech.edu/pub/difmap/difmap2.5e.tar.gz", "457cd77c146e22b5332403c19b29485388a863ec494fff87137176396fc6a9ff"),
+    ArchiveSource("ftp://ftp.astro.caltech.edu/pub/pgplot/pgplot5.2.tar.gz", "a5799ff719a510d84d26df4ae7409ae61fe66477e3f1e8820422a9a4727a5be4")
+]
+
+script = raw"""
+cd $WORKSPACE/srcdir
+mkdir pgplot_build && cd pgplot_build/
+cat ../pgplot/drivers.list | sed 's|! PSDRIV|  PSDRIV|g' | sed 's|! GIDRIV|  GIDRIV|g' > drivers.list
+../pgplot/makemake ../pgplot/ linux g77_gcc
+sed -i 's|FCOMPL=g77|FCOMPL=gfortran|' makefile
+make && make clean
+cd ../uvf_difmap/
+sed -i 's|^PGPLOT_LIB="-lpgplot -lX11"|PGPLOT_LIB="-L/workspace/srcdir/pgplot_build -L/usr/X11R6/lib -Xlinker -R/workspace/srcdir/pgplot_build:/usr/X11R6/lib -lpgplot -lX11"|' configure
+sed -i 's|^USE_TECLA="1"|USE_TECLA="0"|' configure
+./configure linux-i486-gcc
```

Comment:
Why are you passing this argument? This doesn't sound very general

Reply:
This just follows the difmap building instructions (ftp://ftp.astro.caltech.edu/pub/difmap/README). I never built it anywhere outside of x86 linux, not even sure if other systems are well-supported.
They mention in that readme that it should compile on Mac OS as well, but I have no idea how to do that.

Reply:
Inside `configure` there is this comment:
```
# Usage:
#
#  ./configure system_name
#
# Where system_name must be one of the following:
#  sun4-gcc  -  Sun sparc running SUNOS4.* using gcc.
#  sol2-gcc  -  Sun sparc running Solaris 2.* using gcc.
#  sol2-cc   -  Sun Sparc running Solaris 2.* using the SUN ANSI cc
#               compiler.
#  hppa-c89  -  HP9000 700 or 800 series workstations using c89.
#  IBM-c89   -  IBM 6xxx series workstations using c89.
#  alpha-osf1-gcc - Alpha AXP running OSF1, using the Gnu C compiler, gcc.
#  alpha-osf1-cc  - Alpha AXP running OSF1, using the native DEC C compiler.
#  linux-i486-gcc - I486 PC running Linux, using the Gnu C compiler, gcc.
#  linux-ia64-ecc - Itanium PC running Linux, using the Intel compiler ecc.
#  linux-ia64-gcc - Itanium or AMD64 PC running Linux, using gcc.
#  apple-osx-gcc  - Macintosh computer running OSX, using the Gnu C compiler.
#  intel-osx-gcc  - Intel Mac computer running OSX, using the Gnu C compiler.
```

Reply:
Ok, I've read the rest of the script.  I wished I didn't

Reply:
I did't even manage to compile pgplot in the mac environment. It likely requires some source/make file editing.
Changes that may be required to build on linux were known to me beforehand, so I  implemented them with sed here - but have no similar experience with mac.